### PR TITLE
Testsuite: Speedup for sanitizer builds

### PR DESCRIPTION
**Pull request by RudolfWeeber**

### CODE REVIEW

File: testsuite/python/lb_interpolation.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def test_interpolated_velocity(self):
 
         """
         self.set_boundaries()
-        self.system.integrator.run(3000)
+        self.system.integrator.run(1800)
```
Comment (KaiSzuttor) [inline testsuite/python/lb_interpolation.py]: you have to increase this number...

**Comment by fweik** [comment]
@RudolfWeeber can you please address the one remaining problem? The faster we can speed this up the better...

**Comment by RudolfWeeber** [comment]
I'd have expected the lb interpolation test to be deterministic...
I now pushed a version which even passes when the tolerance is lowerd by a factor of 10 - on my machine, that is...

**Comment by fweik** [comment]
It looked deterministic to me: the deviation was exactly the same in all jobs.

**Comment by fweik** [comment]
If PARTICLE_ANISOTROPY is disabled you are comparing a vector to a scalar in `  File "/builds/espressomd/espresso/build/testsuite/python/langevin_thermostat.py", line 145, in test_02__friction_trans`, this is why this test fails with empty config.

**Review by fweik — APPROVED** [review]

**Pull request merged**